File: 第3章 数学广角/3.4 早餐搭配/早餐搭配.py
'''
程序：早餐搭配
作者：苏秦@小海豚科学馆公众号
来源：图书《趣味编程三剑客：从Scratch迈向Python和C++》
'''
def main():
    '''早餐搭配'''
    drinks = ['豆浆', '牛奶']
    foods = ['蛋糕', '油条', '饼干', '面包']
    for j in range(2):
        for i in range(4):
            print(drinks[j], foods[i], sep=' + ')

if __name__ == '__main__':
    main()

File: 第3章 数学广角/3.4 早餐搭配/早餐搭配.cpp
/*
程序：早餐搭配 
作者：苏秦@小海豚科学馆公众号
来源：图书《趣味编程三剑客：从Scratch迈向Python和C++》
*/
#include <bits/stdc++.h>
using namespace std;

// 早餐搭配 
int main()
{
	string drinks[] = {"豆浆", "牛奶"};
	string foods[] = {"蛋糕", "油条", "饼干", "面包"}; 
	for (int j = 0; j < 2; j++)
		for (int i = 0; i < 4; i++)
			cout << drinks[j] << " + " << foods[i] << endl;
	return 0;
}
